pyfstat/make_sfts.py

```diff
@@ -943,6 +941,175 @@ def __init__(
         )
 
 
+class TransientLineWriter(Writer):
+    """Inject a transient line into some data.
+
+    A transient line is defined as a constant amplitude and frequency excess power lasting
+    for a (not necesarilly) finite time span of an observing run.
```

Comment:
```suggestion
    A (transient) line is defined as a constant amplitude and constant frequency excess power artifact,
    lasting for a finite time span or the whole data duration.
```

Reply:
This comment now seems to be orphaned, but if we use `LineWriter` as per above, a similar sentence would fit in well again.

Reply:
Adopted into 89006f7